src/app.rs

```diff
@@ -75,7 +71,7 @@ impl App {
             PathBuf::from(&clashtui_config_dir_str)
         };
 
-        if !clashtui_config_dir.join("basic_clash_config.yaml").exists() {
+        if !clashtui_config_dir.exists(){ //.join("basic_clash_config.yaml").exists() { // weird, shouldn`t we check the dir rather the single file?
```

Comment:
Because of supporting the portable mode, users can simply create a 'data' directory in the program's directory. it would be better to check if the directory is not empty. Checking for file existing is just for convenience, which is not reasonable.